Update service-admin-premium-multi-geo.md (#1205)
MicrosoftDocs/fabric-docs: docs/admin/service-admin-premium-multi-geo.md

* Multi-Geo doesn't support [Metrics in Power BI](/power-bi/create-reports/service-goals-introduction).

> * Workspaces with non Power BI Fabric items can not be moved between regions.  You must delete all the non-Power BI Fabric items before moving a workspace to a different region.  Once the workspace has been moved, it can up to 30 minutes before a non-Power BI items can be created.

Reviewer: ```suggestion
* Workspaces with non-Power BI Fabric items can't be moved between regions.  You must delete all the non-Power BI Fabric items before moving a workspace to a different region.  Once the workspace has been moved, it can take up to 30 minutes before a non-Power BI items can be created.
```

> * Workspaces with non-Power BI Fabric items can't be moved between regions.  You must delete all the non-Power BI Fabric items before moving a workspace to a different region.  Once the workspace has been moved, it can take up to 30 minutes before a non-Power BI items can be created.

Reviewer 2: ```suggestion
* Workspaces with non-Power BI Fabric items can't be moved between regions. You must delete all the non-Power BI Fabric items before moving a workspace to a different region. Once the workspace is moved, it can take up to 30 minutes before non-Power BI items can be created.
```

> * Workspaces with non-Power BI Fabric items can't be moved between regions.  You must delete all the non-Power BI Fabric items before moving a workspace to a different region.  Once the workspace has been moved, it can take up to 30 minutes before a non-Power BI items can be created.

Reviewer 3: ```suggestion
* Workspaces with non-Power BI Fabric items can't be moved between regions.  You must delete all the non-Power BI Fabric items before moving a workspace to a different region.  Once the workspace has been moved, it can take up to 30 minutes before a non-Power BI item can be created.
```
